ionoscloud/resource_k8s_node_pool.go

```diff
@@ -144,10 +144,17 @@ func resourceK8sNodePool() *schema.Resource {
 			"cpu_family": {
 				Type:             schema.TypeString,
 				Description:      "CPU Family",
-				Required:         true,
+				Optional:         true,
 				ForceNew:         true,
 				ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotWhiteSpace),
 			},
+			"server_type": {
+				Type:             schema.TypeString,
+				Description:      "The server type for the compute engine",
+				Optional:         true,
+				Default:          "DedicatedCore",
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice([]string{"DedicatedCore", "VCPU"}, true)),
```

Comment:
@glimberea not sure if we need to ignore case here. I see the enum in the swagger and I suppose that if we send `dedicatedCORE` it should fail, I'm not sure if an exact match is done in the API, so please double check this.